cvat-cli/src/cvat_cli/_internal/commands_tasks.py

```diff
@@ -309,12 +316,19 @@ def execute(
         status_check_period: int,
         include_images: bool,
     ) -> None:
+        if filename is None:
+            filename = os.getcwd()
+
+        if filename[-1:] in ("/", "\\") and not os.path.exists(filename):
+            os.makedirs(filename, exist_ok=True)
```

Comment:
Perhaps we should move this logic to the SDK function? It seems strange to only support automatic directory creation in the CLI.

Reply:
I'm not sure it's a good behavior for an sdk function. Typically we don't do this on this level. For CLI, it reduces the amount of extra typing for the user, so I think it should be done just there.

Reply:
Alright.